app/models/developer.rb

```diff
@@ -14,12 +14,7 @@ class Developer < ApplicationRecord
 
   FEATURE_LENGTH = 1.week
 
-  enum search_status: {
```

Comment:
I was getting warnings about keyword arguments for enums being deprecated in Rails 8.

Reply:
What's the formatting look like if you move each case to its own line? I wonder if that would drastically reduce this PR's diff.

Reply:
I tried that approach and it did reduce the diff.